src/openloco/windows/CompanyWindow.cpp

```diff
@@ -958,6 +958,23 @@ namespace openloco::ui::windows::CompanyWindow
             call(0x00468FFE, regs);
         }
 
+        // 0x00434EC7
+        // input:
+        // regs.esi = window* w;
+        // regs.dx = widgetIndex;
+        // regs.ax = x;
+        // regs.bx = y;
+        // output (not verified):
```

Comment:
I'd prefer you passed in named variables and returned variables rather than passing around registers. I don't think window and widgetIndex are used by this function either. But would need to test that.